ports/certify/portfile.cmake

```
@@ -0,0 +1,11 @@
+
+vcpkg_from_github(
+    OUT_SOURCE_PATH SOURCE_PATH
+    REPO djarek/certify
+    REF a1bd27f2e86ba73d6a64f289f030ec65250edceb
+    SHA512 4fe32f2ae417f5e03cce59ebc0952ae133710c779ab371fa273acade98234a6cb28b55eb1c478a5129939a77489d456cf4ee97ff4dfad42b489e02d6d85a38af
+    HEAD_REF master

```

```suggestion
    REPO  jens-diewald/certify
    REF b09c57ef5f25f4276558376f9c5a02a57b9ded20
    SHA512 998e54fa4c63bb0743137a6f1d9e14c671d82db5bca6cbb1e693b369f348d0f974cdf9516b832ae6fc97164323e769cbe0b122d7938086f8e4d340afe6309890
    HEAD_REF hostname_verification
```

Use the fixed version with proper hostname verification

---

@autoantwort 
Thanks for your help to review. 

I noticed that `jens-diewald/certify` is forked from `djarek/certify`. Generally, we prefer to use the original repo instead of  the forked one.

---

Definitely, but djarek/certify is very inactive and this fork [fixed the hostname verification](https://github.com/djarek/certify/pull/67) (currently broken in djarek/certify, so you can do man in the middle attacks)